xml/obs_source_service.xml

```diff
@@ -207,46 +226,60 @@
  <sect1 xml:id="sec.obs.sserv.validate">
   <title>Defining Services for Validation</title>
   <para>
-   Source Services may be used to validate sources. This can be definied
-   as:
+   Source Services can be used to validate sources. This can be defined
+   at different levels:
   </para>
   <itemizedlist>
    <listitem>
     <formalpara>
      <title>Per Package</title>
      <para>
-      Useful when the packager wants to validate the downloaded sources
-      are really from the original maintainer.
+      Useful when the packager wants to validate wheather the downloaded
+      sources are really from the original maintainer.
      </para>
     </formalpara>
    </listitem>
    <listitem>
     <formalpara>
      <title>Per Project</title>
      <para>
-      Useful for applying general policies which cannot get skipped in
+      Useful for applying project-wide policies which cannot be skipped for
       any package.
      </para>
     </formalpara>
    </listitem>
   </itemizedlist>
-  <remark>toms 2017-08-31: Hmn, not sure how to rephrase the following
-  sentence...</remark>
+
   <para>
-   Validation can happen by validating files, for example, using the
-    <systemitem class="service">verify_file</systemitem> or <systemitem
-      class="service">source_validator</systemitem> service.
-   These services just fail in the error case which leads to the build
-   state "broken". Or validation can happen by redoing a certain action and
-   store the result as new file as <command>download_files</command> is doing.
-   In this case the newly generated file will be used instead of the committed
-   one during build. </para>
+   You can validate sources using either of two methods:
```

Comment:
i am a bit queasy about whether my interpretation of the previous content is correct here -- could you add a fixme remark or so?

Reply:
I've added a remark already in one of the last commits, see the next lines.